extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -94,6 +97,9 @@
 
     SmallRyeGraphQLConfig quarkusConfig;
 
+    @Inject
+    LaunchModeBuildItem launch;
```

Comment:
We tend to avoid that and prefer parameter injection. Otherwise you completely lose the dependencies.

Reply:
OK will change to Parameter.